lab-sarah/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
```

Comment:
You require this in but didn't add it to your package.json, so your gulpfile breaks when you try to use gulp tasks (after pulling down and installing dependencies).